src/main/java/ch/njol/skript/lang/function/FunctionReference.java

```diff
@@ -192,7 +199,7 @@ public boolean validateFunction(boolean first) {
 		}
 
 		// Validate parameter count
-		singleListParam = sign.getMaxParameters() == 1 && !sign.getParameter(0).single;
+		singleListParam = sign.getMaxParameters() == 1 && !sign.parameters().entrySet().iterator().next().getValue().single();
```

Comment:
maybe a helper method? getFirstParam or something?
In general the parameters() method seems to not be very convenient and it's always getting sent to an array or 5 stacked calls or whatever. Perhaps a different choice of data structure would help, or a wrapper around the map that provides helper methods like this